bert/README.md

```diff
@@ -131,59 +136,93 @@ python src/data_c4.py s3://my-bucket/my-copy-c4 /tmp/cache-c4
 
 Now that you've installed dependencies and built a local copy of the C4 dataset, let's start training! We'll start with MLM pre-training on C4.
 
-**Please remember** to edit the `data_remote` and `data_local` paths in your YAML to point to your local C4 copy.
-Our streaming dataloader always streams from `data_remote` -> `data_local`, and if both paths are the same,
-then no extra copying is done.
+**Note:** the YAMLs default to using `./my-copy-c4` as the location of your C4 dataset, following the above examples. **Please remember** to edit the `data_remote` and `data_local` paths in your pre-training YAMLs to reflect any differences in where your dataset lives (e.g., if you used a different folder name or already moved your copy to S3).
 
 **Also remember** that if you only downloaded the `val` split, you need to make sure your train_dataloader is pointed at that split.
 Just change `split: train` to `split: val` in your YAML.
-This is already done in the testing YAML `yamls/test-cpu/main.py`, which you can also use to test your configuration.
+This is already done in the testing YAML `yamls/test/main.py`, which you can also use to test your configuration (see [Quick Start](#quick-start)).
 
 ## MLM pre-training
 
-We run the `main.py` script using our `composer` launcher, which generates N processes (1 process per GPU device).
+To get the most out of your pre-training budget, we recommend using **Mosiac BERT**! You can read more [below](#mosaic-bert).
 
-If training on a single node, the `composer` launcher will autodetect the number of devices, so all you need to do to pre-train a Mosaic BERT is run:
+We run the `main.py` pre-training script using our `composer` launcher, which generates N processes (1 process per GPU device).
+If training on a single node, the `composer` launcher will autodetect the number of devices.
 
 ```bash
-composer main.py yamls/mosaic-bert-base-uncased.yaml
-```
+# This will pre-train a HuggingFace BERT that reaches a downstream GLUE accuracy of about 83.3%.
+# It takes about 11.5 hours on a single node with 8 A100_80g GPUs.
+composer main.py yamls/main/hf-bert-base-uncased.yaml
 
-**Note:** The `yamls/*.yaml` files are intended to be used with `main.py`.
+# This will pre-train a Mosaic BERT that reaches the same downstream accuracy in roughly 1/3 the time.
+composer main.py yamls/main/mosaic-bert-base-uncased.yaml
+```
 
-**Please remember** to modify the reference YAMLs (e.g., `yamls/mosiac-bert-base-uncased.yaml`) to customize saving and loading locations -- only the YAMLs in `yamls/test-cpu/` are ready to use out-of-the-box.
+**Please remember** to modify the reference YAMLs (e.g., `yamls/main/mosiac-bert-base-uncased.yaml`) to customize saving and loading locations—-only the YAMLs in `yamls/test/` are ready to use out-of-the-box. See the [configs](#configs) section for more detail.
```

Comment:
```suggestion
**Please remember** to modify the reference YAMLs (e.g., `yamls/main/mosaic-bert-base-uncased.yaml`) to customize saving and loading locations—only the YAMLs in `yamls/test/` are ready to use out-of-the-box. See the [configs](#configs) section for more detail.
```
The first dash is already an em dash; having both looks weird in the rendered readme